File: mainActivity/src/main/java/com/sierrawireless/avphone/MonitorServiceManager.java
package com.sierrawireless.avphone;

import com.sierrawireless.avphone.service.MonitoringService;


public interface MonitorServiceManager {
    boolean isServiceRunning();
    boolean isServiceStarted(String name);
    boolean oneServiceStarted();
    void stopMonitoringService();
    void startMonitoringService(String name);
    void startSendData();
    void stopSendData();
    void sendAlarmEvent();
    void setMonitoringServiceListener(MonitorServiceListener listener);
    MonitoringService getMonitoringService();
}

File: mainActivity/src/main/java/com/sierrawireless/avphone/MonitorServiceManager.kt
package com.sierrawireless.avphone

import com.sierrawireless.avphone.service.MonitoringService


interface MonitorServiceManager {
    //val isServiceRunning: Boolean
    var monitoringService: MonitoringService?
    fun isServiceStarted(name: String): Boolean
    fun oneServiceStarted(): Boolean
    fun stopMonitoringService()
    fun startMonitoringService(name: String)
    fun startSendData()
    fun stopSendData()
    fun sendAlarmEvent()
    fun setMonitoringServiceListener(listener: MonitorServiceListener)
    fun isServiceRunning(): Boolean
}